tests/test.bats

```diff
@@ -7,7 +7,7 @@ function wait_for () {
 
   local hostname=$1
   local port=$2
-  local retries=10
+  local retries=20
```

Comment:
Why do we need to increase the timeout... Is the app getting slower?

Reply:
I don't know what is happening that causes the occasional timeouts. Rerunning usually succeeds.

```
ERROR: for app  UnixHTTPConnectionPool(host='localhost', port=None): Read timed out. (read timeout=60)
ERROR: An HTTP request took too long to complete. Retry with --verbose to obtain debug information.
If you encounter this issue regularly because of slow network conditions, consider setting COMPOSE_HTTP_TIMEOUT to a higher value (current value: 60).
make: *** [Makefile:30: up-with-data] Error 1
```

Reply:
I think we need to set the COMPOSE_HTTP_TIMEOUT to something higher than the default of 60 seconds. Retrying a nc wait after the docker-compose timeout has happened won't be successful.

Reply:
Timeout was bumped due to pip installation of mapped volumes in the start script (ln 53). The mapping is much more targeted now and should no longer hit above the limit unless multiple/all extensions are mapped. Leaving retries at 20 just in case.